### ISSUE TITLE
Narrator Reads out Non Active Fields on Npm Install Window

### ISSUE THREAD

**Issue by mjbvz** [Microsoft/nodejstools#1357]
##### Repo
1. Enable windows narrator mode.
2. Open the NTVS npm install window.
3. Wait for the window to load fully.
4. Press Caps Lock + w to read the window.

Here's the window:

![image](https://cloud.githubusercontent.com/assets/12821956/19531516/d8906396-95ec-11e6-9b59-5426c53eead7.png)
##### Expected

Reads out only currently active content of the window
##### Actual

Narrator starts reading through the elements of the window, but also lists multiple "0% progress bar" and similar elements. The progress bars are used when actually installing a package and are not visible/active at this point, and therefore should not be read to the user.

**Comment by paulvanbrenk** [comment]
Fixed in Update 3
